src/scenes/InternalScene.ts

```diff
@@ -17,72 +19,149 @@ export default class InternalScene extends BaseScene {
     }
 
     setup(){
+        this.macrophageArray.forEach(macrophage => {
+            macrophage.gotoAndStop(0);
+            macrophage.velocity = new PIXI.Point(Math.random()*.5, Math.random()*.5);
+        })
 
+        this.bacteriaArray.forEach(bacteria => {
+            bacteria.gotoAndStop(0);
+            bacteria.velocity = new PIXI.Point(Math.random(), Math.random());
+        })
```

Comment:
move lines below the lines that fill the array (line 34 in this commit) ! The scene is hanging because the velocity is not set.